coordinator/src/routes.rs

```diff
@@ -380,6 +387,25 @@ pub async fn get_user(
     }
 }
 
+#[instrument(skip_all, err(Debug))]
+pub async fn get_user_referral_status(
+    State(state): State<Arc<AppState>>,
+    Path(trader_pubkey): Path<String>,
+) -> Result<Json<commons::ReferralStatus>, AppError> {
+    let mut conn = state
```

Comment:
🔧 we should verify the users intend on that api. Otherwise anybody could query the referral status of any user they know the pubkey of.

Reply:
I removed the endpoint and return the referral status from with the authenticated response from the websocket.